### ISSUE TITLE
Double fault measure

### ISSUE THREAD

**Issue by alexrame** [google/uncertainty-metrics#62]
Hello, thanks for the nice work
I think there is a mistake in the double fault measure of diversity in
https://github.com/google/uncertainty-metrics/blob/master/uncertainty_metrics/tensorflow/diversity.py

The current code:
```
double_faults = tf.cast(double_faults, tf.float32)
return tf.reduce_mean(double_faults)
```
computes the probability of logits 2 to be wrong conditionally to logits 1 being wrong.
The double fault measure the probability that both are wrong simulatenously.
Therefore, one possible solution is:

```
double_faults = tf.cast(double_faults, tf.float32)
return tf.reduce_sum(double_faults) / preds_1.shape[0]
```

**Comment by JeremyNixon** [comment]
Hello Alex! Thank you for the suggestion - #64 is under review.
